### PR TITLE
Update Test structure

### PR DESCRIPTION
**Pull request by amityadav0**
- Use sdk sim.

### CODE REVIEW

File: nearswap/tests/nep21_transfer.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#![allow(unused)]
```
Comment (robert-zaremba) [inline nearswap/tests/nep21_transfer.rs:1]: Why not moving this this to `other-contracts/nep-21/test`?

File: nearswap/tests/simulation_test.rs [hunk 2]
```diff
@@ -3,162 +3,47 @@
...
+use near_sdk_sim::{
+    call, deploy, init_simulator, near_crypto::Signer, to_yocto, view, ContractAccount,
+    UserAccount, STORAGE_AMOUNT,
+};
+use near_sdk_sim::account::AccessKey;
 
 use nearswap::util::*;
 use nearswap::PoolInfo;
-
+use nearswap::NearCLPContract;
```
Comment (robert-zaremba) [inline nearswap/tests/simulation_test.rs:17]:
```suggestion
use nearswap::{NearCLPContract, PoolInfo};
```
And remove the line before.

File: nearswap/tests/nep21_transfer.rs (hunk 1 above)
Comment (robert-zaremba) [reply nearswap/tests/nep21_transfer.rs:1]: I see, we move it from the simulation_test.rs. Let's move it to the location suggested above.

File: nearswap/tests/test_utils.rs [hunk 3]
```diff
@@ -1,225 +1,196 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // Copyright (C) 2020 Robert Zaremba and contributors
 
-use nearswap::util::*;
-use near_primitives::types::{AccountId, Balance};
+#![allow(unused)]
+
```
Comment (robert-zaremba) [inline nearswap/tests/test_utils.rs:5]: let's:
* rename this file to `clp_utils.rs`
* move all not clp directly related things (eg `get_nep21_balance` , `show_nep21_bal
`) to other files (eg: `nep21_utils.rs`)

**Review by robert-zaremba — COMMENTED** [review]
This looks good, left few comments.

File: nearswap/Cargo.toml [hunk 4]
```diff
@@ -23,6 +23,7 @@ funty = "=1.1.0"
 #New version of simulation tests:
 # [dev-dependencies]
 near-sdk-sim = { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
```
Comment (robert-zaremba) [inline nearswap/Cargo.toml:25]:
```suggestion
near-sdk-sim = "=3.0.0-pre.2" // { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
```

File: nearswap/tests/nep21_transfer.rs (hunk 1 above)
Comment (amityadav0) [reply nearswap/tests/nep21_transfer.rs:1]: OK

File: nearswap/tests/simulation_test.rs (hunk 2 above)
Comment (amityadav0) [reply nearswap/tests/simulation_test.rs:17]: ok

File: nearswap/tests/test_utils.rs (hunk 3 above)
Comment (amityadav0) [reply nearswap/tests/test_utils.rs:5]: ok

File: nearswap/Cargo.toml (hunk 4 above)
Comment (amityadav0) [reply nearswap/Cargo.toml:25]: Screenshot from 2021-03-04 22-16-56
this is not working

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**